npm-debug.log

```diff
@@ -0,0 +1,75 @@
+0 info it worked if it ends with ok
```

Comment:
??

Reply:
Sorry, I'll remove that. It got generated on its own somehow